Jenkinsfile

```diff
@@ -152,10 +153,13 @@ stage ("discover tests") {
 				}
 
 				def isAllowedVersion = jobCfg.options?.allowedOrchestratorVersions == null ? true : version in jobCfg.options.allowedOrchestratorVersions
+				isAllowedVersion |= (version.equals(latestReleasedVersion)) && jobCfg.options?.allowedOrchestratorVersions != null && "latestReleasedVersion" in jobCfg.options.allowedOrchestratorVersions

```

Comment:
I think we can clean this up a bit.
```groovy
nullOps = null
fooOps = ["foo"]
blahOps = ["blah"]
foo = "foo"

println(nullOps && "blah" in nullOpts)   // false
println(fooOps && "blah" in fooOps)      // false
println(blahOps && "blah" in blahOps)    // true
println(nullOps ?: "foo")                // foo
println(fooOps[0] == foo)                // true

isTrue = false
isTrue |= foo == "foo" && fooOps && foo in fooOps
println(isTrue)                          // true

// check out https://groovy-playground.appspot.com
```
So...
```groovy
def allowedVersions = jobCfg.options?.allowedOrchestratorVersions
def isAllowedVersion = allowedVersions ?: version in allowedVersions
isAllowedVersion |= version == latestReleasedVersion && allowedVersions && "latestReleasedVersion" in allowedVersions
```

Reply:
i'll try that out. I'm a total noob w.r.t. js/java/groovy/whatever this is :P

Reply:
updated.
I needed to change the second line to isAllowedVersions = allowedVersions == null ? true : version in allowedVersions because ?: assigned the value of allowedVersions to isAllowedVersions if it is not null.